RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -6451,7 +6451,7 @@
 				SUPPORTED_PLATFORMS = "appletvos appletvsimulator iphoneos iphonesimulator macosx watchos watchsimulator xros xrsimulator";
 				SUPPORTS_MACCATALYST = YES;
 				SUPPORTS_XR_DESIGNED_FOR_IPHONE_IPAD = YES;
-				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = "DEBUG CUSTOMER_CENTER_ENABLED";
```

Comment:
@tonidero I found that this additional change was needed to pick up the flag in `RevenueCat` (as opposed to `RevenueCatUI`). To be specific: `CustomerCenterConfigData` and `Purchases.loadCustomerCenter()` could not be resolved without this change. 

Apologies for force pushing to your branch, but I decided its best to amend your commit than to make an additional commit. Otherwise we'd have to cherry-pick and revert 2 commits each time during development.

Reply:
Ah I might have missed it. Thanks!